src/assets.zig

```diff
@@ -157,6 +149,13 @@ pub fn readAllZonFilesInAddons(
 				output.put(id, zon) catch unreachable;
 			}
 		}
+		if(migrations != null) blk: {
+			const zon = main.files.Dir.init(dir).readToZon(externalAllocator, "_migrations.zig.zon") catch |err| {
+				if(err != error.FileNotFound) std.log.err("Missing {s} migration file for addon {s}", .{subPath, addonName});
```

Comment:
The error is (and should only be) printed when there is an error reading the file (such as AccessDenied).
So the message should be "Cannot read {s} migration file for addon {s}"